src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -3564,6 +3573,42 @@ instruct vmask_reinterpret_diff_esize(vRegMask dst, vRegMask_V0 src, vReg tmp) %
   ins_pipe(pipe_slow);
 %}
 
+// ------------------------------ Vector shuffle -------------------------------
+
+instruct loadshuffleB(vReg dst) %{
+  predicate(Matcher::vector_element_basic_type(n) == T_BYTE);
+  match(Set dst (VectorLoadShuffle dst));
+  effect(TEMP_DEF dst);
```

Comment:
Seems no need to add a `TEMP_DEF` for dst here.

Reply:
fixed, thanks for catching.